src/notification.c

```diff
@@ -47,130 +48,444 @@ struct _notification_private {
         gint refcount;
 };
 
+extern char **environ;
+
+
+//TODO: Move to utils.c but for now I wanna edit only the files I have to...
```

Comment:
OK, will move it in the next commit. I was pushing most of the changes to one file, because I was not sure if you would want to merge it :slightly_smiling_face:

Reply:
OK, will move it in the next commit. I was pushing most of the changes to one file, because I was not sure if you would want to merge it :slightly_smiling_face: